src/mongo/unittest/unittest.cpp

```diff
@@ -159,6 +169,56 @@ void Test::run() {
 void Test::setUp() {}
 void Test::tearDown() {}
 
+//
+void Benchmark::run() {
+
+    BenchmarkStats stats = BenchmarkStats();
+    stats.name = testName();
+
+    float totalOpsPerSec = 0;
+
+    for (int r = 0; r < RUNS; r++) {
+
+        setUp();
+
+        Date_t start_time = Date_t::now();
```

Comment:
Use camelCase thoughout. Please check everything in this commit and fix anything that isn't named correctly.